app/src/main/java/com/intern/conjob/ui/auth/login/LoginFragment.kt

```diff
@@ -55,19 +55,14 @@ class LoginFragment : BaseFragment(R.layout.fragment_login) {
                 ).onSuccess {
                     (activity as OnBoardingActivity).startActivity(Intent(context, MainActivity::class.java))
                     (activity as OnBoardingActivity).finish()
-                }.onError(
-                    commonAction = {
+                }.onError{
```

Comment:
onError(
                    commonAction = {
                        viewModel.emitErrorModel(it)
                    },
                    normalAction = {
                        tvLoginValidate.visibility = View.VISIBLE
                        tvLoginValidate.text = it.message
                    }
                )